### ISSUE TITLE
Add ability to remove a spinner from the spinners list

### ISSUE THREAD

**Issue by jcarpanelli** [jcarpanelli/spinnies#5]
The idea is to implement a new method called `remove` that receives a spinner reference name as argument and removes that spinner from the list, and thus should not be printed again in the terminal If no reference is given, the method call does nothing.

Example usage:
```js
const spinnies = new Spinnies();
spinnies.add('spinner-1');
spinnies.add('spinner-2');
spinnies.add('spinner-3');
setTimeout(() => {
  spinnies.remove('spinner-2');
}, 3000); // => should remove the second spinner and remove it from `stderr`. 
```

**Comment by SweetMNM** [comment]
I implemented this feature in my [fork](https://github.com/SweetMNM/spinnies).
I'm working on more features, then i will open a PR.

**Comment by jcarpanelli** [comment]
Hey @SweetMNM! Thanks for tackling this! Looking forward to reviewing that PR :slightly_smiling_face:

**Comment by jcarpanelli** [comment]
Hey @SweetMNM, I noticed you have been improving `spinnies` a lot in your fork! Do you plan to create a PR soon?

**Comment by SweetMNM** [comment]
Yes, I just want to finish implementing custom statuses, hide and unhide spinners, write some more tests and maybe some other features before I open the PR..
